Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -17,11 +17,14 @@ import ArcGIS
 
 @MainActor
 @preconcurrency
-public struct PreplannedListItemView: View {
+struct PreplannedListItemView: View {
     /// The view model for the preplanned map.
     @ObservedObject var model: PreplannedMapModel
     
-    public var body: some View {
+    /// The closure to perform when the map selection changes.
+    var onMapSelectionChanged: ((Map) -> Void)?
```

Comment:
This can be non-optional and immutable, correct?

Reply:
Yes.

Reply:
Should it be `var onMapSelectionChanged: ((Map) -> Void)? = nil` with a default value, so a user doesn't have to provide a closure? I imagine that would be a rare case though, as one of the goals of this component is to allow a user to get the selected map.

Reply:
No, because this is an internal property and a closure is always provided.

Reply:
That's right, I misread it 🤕